docs/src/pages/[platform]/connected-components/authenticator/quick-start-example.angular.mdx

```diff
@@ -45,6 +45,27 @@ Wrap your template with `amplify-authenticator` inside `app.component.html`
   </ng-template>
 </amplify-authenticator>
 ```
+**Only if you are using Angular 17,** import the `AmplifyAuthenticatorModule` in `app.component.ts`
+
+```ts{3,8,9}
+import { Component } from '@angular/core';
+import { Amplify } from 'aws-amplify';
+import { AmplifyAuthenticatorModule, AuthenticatorService } from '@aws-amplify/ui-angular';
+
+import awsExports from './aws-exports';
+
+@Component({
+  standalone: true,
+  imports: [CommonModule, RouterOutlet, AmplifyAuthenticatorModule],
```

Comment:
Not sure if it's misleading to have all three imports when AmplifyAuthenticatorModule is the only new one that should be added

Reply:
But I thought it might also be confusing to not put the other imports because they're used in the advanced usage example

Reply:
I think it makes sense given angular users will probably already be familiar with the others.